src/snarks/groth16/evaluator_from_lagrange.tcc

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "evaluator_from_lagrange.hpp"
+#include "multi_exp.hpp"
+
+namespace libzeth
+{
+
+template<typename ppT, typename GroupT>
+evaluator_from_lagrange<ppT, GroupT>::evaluator_from_lagrange(
+    const std::vector<GroupT> &powers,
+    libfqfft::evaluation_domain<libff::Fr<ppT>> &domain)
+    : powers(powers), domain(domain)
+{
+    assert(powers.size() >= domain.m - 1);
```

Comment:
Shouldn't it be a strict inequality here?
The lagrange polynomials with regard to the domain S have a degree bound by `|S| - 1`. Hence, they have at most |S| terms, and each of this term represents a power of `x`, `x^i` for `i \in {0, ..., |S| - 1}`. Maybe I am missing something @dtebbs ?

Reply:
Yes.  Fixed and added a comment.  Thanks.